tests/test_F0_output.py

```diff
@@ -70,6 +91,7 @@ def setup_class(self):
     def test_store_barchart_for_capacities_no_additional_capacities(self):
         """ """
         dict_scalar_capacities = {
+            PATHS_TO_PLOTS: dict_plots[PATHS_TO_PLOTS],
```

Comment:
You can use `dict_scalar_capacities.update(dict_plots)` after `dict_scalar_capacities` is created

Reply:
True xD
Maybe I should think more when I am fixing stuff.